spec/dummy/config/application.rb

```diff
@@ -23,9 +23,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
-
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
We probably should really set up something like Appraisal. I added an issue #10

It's probably going to be a little bit of work at least because we'll still only have 1 dummy app. Unless we change that too. Anyway, beyond the scope of this PR.